lib/eblox/data/post.ex

```diff
@@ -86,22 +86,20 @@ defmodule Eblox.Data.Post do
     end
   end
 
-  @behaviour Siblings.Worker
-
-  @impl Siblings.Worker
-  def perform(:idle, _id, _payload) do
-    {:transition, :read, nil}
+  @impl Finitomata
+  @doc false
+  def on_transition(:parsed, :update, event_payload, payload) do
+    {:ok, :parsed, Map.put(payload, :data, event_payload)}
   end
 
-  @impl Siblings.Worker
-  def perform(:read, _id, _payload) do
-    {:transition, :parse, nil}
-  end
+  @behaviour Siblings.Worker
 
+  @impl Siblings.Worker
   def perform(:parsed, _id, _payload) do
-    {:reschedule, @interval_after_parse}
+    :noop
```

Comment:
Thanks to auto transitions, which are immediate, we don’t need rescheduling anymore.